hw-model/src/model_emulated.rs

```diff
@@ -54,7 +54,7 @@ impl<'a> Bus for EmulatedApbBus<'a> {
 
 /// Emulated model
 pub struct ModelEmulated {
-    cpu: Cpu<BusLogger<CaliptraRootBus>>,
+    cpu: Box<Cpu<BusLogger<CaliptraRootBus>>>,
```

Comment:
To avoid this problem in the future or for other types, would it be better to modify the `HwModel` trait to return a `Box`? I'm not sure how easy that is without knowing the concrete type of `Self` though...

Reply:
ModelEmulated is a fairly light wrapper around Cpu<CaliptraRootBus>, which does all the heavy lifting. Returning a Box would arguably be a bit better, but I don't want to change the HwModel's public API, which has hundreds of tests written against it today and is also used in vendor code.

If we did want to go that route, I would do something like this and then forward all the functions:

pub struct ModelEmulated {
  imp: Box<ModelEmulatedImpl>,
}